scripts/unsubscribe_inactive_people.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+def unsubscribe
+  User.all.each do |user|
+    next unless user.user_contests.order(:created_at).last.created_at < Time.current - 6.months
+
+    user.user_notifications.each(&:destroy!)
+    Mailgun.send_message(
+      to: user.email,
+      text: 'Kami melihat bahwa Anda sudah tidak mengikuti KTOM selama 6 bulan terakhir.' \
```

Comment:
Can you put the spaces at the back? So that it is consistent with other multiline strings.

Also, change the word "stop" to "berhenti".

Otherwise, this looks good!